db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_04_25_072002) do
+ActiveRecord::Schema.define(version: 2022_05_25_101004) do
```

Comment:
@developedBySJ 
Is this by mistake?

Reply:
@mabras Schema file is generated when I applied the latest migrations.